firestore/FirestoreSwiftUIExample/Views/StarsView.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  StarsView.swift
+//  FirestoreSwiftUIExample
+//
+
+import SwiftUI
+
+struct StarsView: View {
+    var rating: Int
+
+    var body: some View {
+        HStack {
+            ForEach(1..<6) { i in
+                getStar(num: i, rating: rating)
+                    .frame(width: 12, height: 20)
+                    .foregroundColor(getStarColor(num: i, rating: rating))
+            }
+        }
+    }
```

Comment:
Wow, this is way more concise than the old implementation :)

Reply:
SwiftUI FTW :-)